From 89efbb13060c483639ce94edeba47c4fcdcad388 Mon Sep 17 00:00:00 2001
From: Sai Lakshmi Teja Vanka <SaiLakshmiTeja.Vanka@windriver.com>
Date: Fri, 18 Jul 2025 09:48:51 +0000
Subject: [PATCH] Remove log_dir from conf files

Adapt 0001-Remove-log_dir-from-conf-files.patch from CentOS.

Signed-off-by: Sai Lakshmi Teja Vanka <SaiLakshmiTeja.Vanka@windriver.com>
---
 manifests/log.pp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/manifests/log.pp b/manifests/log.pp
index 3f7f4e1..4811284 100644
--- a/manifests/log.pp
+++ b/manifests/log.pp
@@ -30,6 +30,7 @@
 #   (Optional) Directory where logs should be stored.
 #   If set to $facts['os_service_default'], it will not log to any directory.
 #   Defaults to $facts['os_service_default']
+#   STX: Remove log_dir to ensure services log via syslog
 #
 # [*use_syslog*]
 #   (Optional) Use syslog for logging (boolean value).
@@ -164,7 +165,7 @@ and will be removed in a future release.")
     'DEFAULT/log_config_append'             => { value => $log_config_append },
     'DEFAULT/log_date_format'               => { value => $log_date_format },
     'DEFAULT/log_file'                      => { value => $log_file },
-    'DEFAULT/log_dir'                       => { value => $log_dir },
+    'DEFAULT/log_dir'                       => { ensure => absent },
     'DEFAULT/watch_log_file'                => { value => $watch_log_file_real },
     'DEFAULT/use_syslog'                    => { value => $use_syslog },
     'DEFAULT/use_journal'                   => { value => $use_journal },
-- 
2.47.2

